com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## [2.2.0-exp.3] - 2021-01-22
 ### Changes
 - Minimum Unity version is 2019.4
+- Fixed a memory leak when using varying topology meshes.
```

Comment:
Suggestion:
`- Fixed a memory leak that occurred when using varying topology meshes.`
Also move the bullet to "Fixed" section.